crates/starknet-server/src/server.rs

```diff
@@ -37,6 +37,7 @@ pub fn serve_http_api_json_rpc(
             post(http::postman::postman_consume_message_from_l2),
         )
         .http_api_route("/create_block", post(http::blocks::create_block))
+        .http_api_route("/get_latest_block", post(http::blocks::get_latest_block))
```

Comment:
Still not done with the review, but I'll just leave this comment not to forget tomorrow.
I don't see a need for this specific endpoint. Even if RPC method `starknet_getBlock` doesn't support getting the latest block, we should be able to retrieve any block by its hash and I assume in the case where you "need" this endpoint, you also know the block hash.

Reply:
Fixed in 48015d28db3fce9bfd6924c695e918ad7ae342a7